### PR TITLE
Roster remove participant

### PR DESCRIPTION
**Pull request by adamweeks**
participant-add-remove

### CODE REVIEW

**Comment by adamweeks** [comment]
@bzang ready for review again!

**Comment by adamweeks** [comment]
participant-add-remove

Updated UX

File: packages/node_modules/@ciscospark/react-redux-spark/src/__snapshots__/reducer.test.js.snap [hunk 1]
```diff
@@ -2,58 +2,58 @@
...
 exports[`spark reducer should handle UPDATE_SPARK_STATUS 1`] = `
 Immutable.Map {
-  "error": null,
-  "status": Immutable.Map {
-    "authenticated": true,
-    "authenticating": false,
-    "registered": true,
-    "registerError": false,
-    "registering": false,
+  error: null,
```
Comment (bzang) [inline packages/node_modules/@ciscospark/react-redux-spark/src/__snapshots__/reducer.test.js.snap]: This doesn't look quite right.... There was a change in jest or Immutable that moved from no quotations around key names to with quotations. Can you check to make sure you're up to date with jest and immutable and run `jest` again?

Comment (adamweeks) [reply packages/node_modules/@ciscospark/react-redux-spark/src/__snapshots__/reducer.test.js.snap]: Forgot to `npm i` after merging. Fixing now.

**Comment by adamweeks** [comment]
@bzang snapshots fixed

**Pull request merged**